### PR TITLE
Adds donor items, fixes some donor unders.

### PR DESCRIPTION
**Pull request by pireamaineach**
## Changelog
:cl:
add: Adds the mountaineer's knapsack and clothing, for myself.
fix: adds _s to several donor unders' on_mobs, the lack there of previously caused them to be invisible.
/:cl:

### CODE REVIEW

File: modular_citadel/code/game/objects/items/weapons/storage/donatorbackpack.dm [hunk 1]
```diff
@@ -14,4 +14,11 @@
...
-	item_state = "cameronbackpack"
\ No newline at end of file
+	item_state = "cameronbackpack"
+
+/obj/item/weapon/storage/backpack/satchel/donator/knapsack
+	name = "mountaineer's knapsack"
+	desc = "A rustic cotton knapsack with leathre straps. It's got a cozy looking blanket strapped to it."
+	icon_state = "knapsack"
+	item_state = "knapsack"
+	starts_with = list(/obj/item/clothing/shoes/boots/winter/explorer,	/obj/item/clothing/suit/storage/hooded/wintercoat/medical/sar)
```
Comment (LetterJay) [inline modular_citadel/code/game/objects/items/weapons/storage/donatorbackpack.dm]: remove this line

Comment (pireamaineach) [reply modular_citadel/code/game/objects/items/weapons/storage/donatorbackpack.dm]: Sad!

Comment (pireamaineach) [reply modular_citadel/code/game/objects/items/weapons/storage/donatorbackpack.dm]: Not going to question it I suppose. Though I don't think that offers any mechanical advantage.

**Comment by VailTheWolf** [comment]
Aren't you banned right now?

**Comment by pireamaineach** [comment]
No

**Pull request closed without merging**